### DISCUSSION TITLE
Bringing back the Stage3D/Flash Online Conference

### DISCUSSION THREAD

**Discussion by Fancy2209** [airsdk/Adobe-Runtime-Support#3336] [General]
There used to be a yearly online conference hosted by the community where Adobe and Library Developers would talk about what they are currently working on, but unfortunately they stopped. 
I created this thread to stir up discussion to see if people are still interested in something of the kind, and also to discuss the feasibility of doing it nowdays

**Comment by marchbold** [comment]
I don't think I remember any online conferences, but that definitely sounds like a good idea. I managed to make it to "flash on the beach" one year which was such an amazing event. So full of creativity and passion.
